### DISCUSSION TITLE
Function not rendering issue on Wifi

### DISCUSSION THREAD

**Discussion by seanspeng** [SilentVoid13/Templater#1157] [Help] [answered]
Strange rendering issue

I'm experiencing very strange rendering issue:

While on Wi-Fi, some contents do not render - see red underlined. 

If I switch to cell phone LTE connection, then they render correctly. 

Obsidian benefit is local, and write any time. I noticed that if there is no internet at all, these functions do not render either. 

Anyone can give me some pointer?

![IMG_4277](https://github.com/SilentVoid13/Templater/assets/137311171/775b7936-bb2f-4795-93d5-84e13750ced3)

**Comment by Zachatoo** [comment, marked as answer]
Try removing the daily quote part. I'm curious how it works at all, given this bug report, but maybe it only works on cellular?

https://github.com/SilentVoid13/Templater/issues/1035

Either way, getting the daily quote requires an Internet connection, if you don't want to require an Internet connection, you should remove it.

And, usually if one part of the template fails, the whole template fails.

**Reply by seanspeng** [reply]
That's right, removing the daily quote fixed the issue. Thank you!

**Comment by AutonomousWork** [comment]
Strangely enough, I have the same issue. I cannot render daily quote on WiFi (mobile or desktop). The other variables do render on WiFi and the only resolution for quote is to use cellular. 

![image](https://github.com/SilentVoid13/Templater/assets/134229312/b0cd2a1a-dc03-44b4-a5f5-f8fee8c3817c)
